### PR TITLE
node-fs-readfile - Joseph Lee

### PR DESCRIPTION
**Pull request by leejaejoseph**
node-fs-readfile

### CODE REVIEW

File: node-fs-readfile/read-any.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+const reference = process.argv[2];
```
Comment (RobertGardner) [inline node-fs-readfile/read-any.js:1]: I like that you do this in a separate variable. Note, though, that by convention, the `require` statements always go at the top. When we do `import/export`, the `import` will be required to be first.

File: node-fs-readfile/read-any.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+
+const fs = require('fs');
+fs.readFile(reference, 'utf8', (err, data) => {
+  if (err) throw err;
```
Comment (RobertGardner) [inline node-fs-readfile/read-any.js:5]: Good! Have you tried this with a non-existent file so you can what it prints?

**Review by RobertGardner — CHANGES_REQUESTED** [review]

File: node-fs-readfile/read-any.js (hunk 2 above)
Comment (leejaejoseph) [reply node-fs-readfile/read-any.js:5]: Yes I have. It returns an error of ENOENT: no such file or directory, open (missing file)

File: node-fs-readfile/read-any.js (hunk 1 above)
Comment (leejaejoseph) [reply node-fs-readfile/read-any.js:2]: Sounds good

**Review by RobertGardner — APPROVED** [review]

**Pull request merged**